apps/worker/src/app/workflow/usecases/run-job/run-job.usecase.ts

```diff
@@ -73,11 +71,15 @@ export class RunJob {
 
       await this.storageHelperService.getAttachments(job.payload?.attachments);
 
-      const template = await this.getNotificationTemplate({
-        _id: job._templateId,
-        environmentId: job._environmentId,
+      const notification = await this.notificationRepository.findOne({
```

Comment:
small question, right now the template was used only for tags?

Reply:
Yes, exactly. Paweł added this lookup ~1 month ago to support this feature and it's not used in Production today - therefore no risk of breaking anything.